composer.json

```diff
@@ -34,10 +34,5 @@
         "phpunit/phpunit": "^4.5 || ^5.0.5",
         "phpunit/phpunit-mock-objects": "2.3.0 || ^3.0",
         "mikey179/vfsStream": "^1.6"
-    },
-    "config": {
-        "platform": {
-            "php": "5.3.3"
```

Comment:
Remove this entirely is wrong, as it means that the Satis phar won't be compatible with PHP 5.3 anymore (your lock file update bumped the requirement to PHP 5.5.9 as it uses Symfony 3)